jormungandr/src/explorer/graphql/mod.rs

```diff
@@ -46,10 +46,13 @@ impl Block {
     }
 
     fn get_explorer_block(&self, db: &ExplorerDB) -> FieldResult<ExplorerBlock> {
-        db.get_block(&self.hash).wait()?.ok_or(
-            ErrorKind::InternalError("Couldn't find block's contents in explorer".to_owned())
-                .into(),
-        )
+        db.get_block(&self.hash)
+            .wait()
+            .unwrap_or_else(|e| match e {})
```

Comment:
To be replaced by `.into_ok()` when that is stabilized: https://github.com/rust-lang/rust/pull/66045